snow/validators/manager.go

```diff
@@ -89,69 +298,21 @@ func (m *manager) String() string {
 		sb.WriteString(fmt.Sprintf(
 			"\n    Subnet[%s]: %s",
 			subnetID,
-			vdrs.PrefixedString("    "),
+			vdrs.PrefixedString(""),
```

Comment:
Why did we remove the prefix?

Reply:
We already have 4 spaces prefix in set. So we endup having 8 spaces prefix. I can restore it but this seems cleaner.

See current test string in `TestString` test:
```go
	expected := "Validator Manager: (Size = 2)\n" +
		"    Subnet[TtF4d2QWbk5vzQGTEPrN48x6vwgAoAmKQ9cbp79inpQmcRKES]: Validator Set: (Size = 2, Weight = 9223372036854775807)\n" +
		"    Validator[0]: NodeID-111111111111111111116DBWJs, 1\n" +
		"    Validator[1]: NodeID-QLbz7JHiBTspS962RLKV8GndWFwdYhk6V, 9223372036854775806\n" +
		"    Subnet[2mcwQKiD8VEspmMJpL1dc7okQQ5dDVAWeCBZ7FWBFAbxpv3t7w]: Validator Set: (Size = 1, Weight = 1)\n" +
		"    Validator[0]: NodeID-QLbz7JHiBTspS962RLKV8GndWFwdYhk6V, 1"
```

Reply:
We should restore it imo.
```
Validator Manager: (Size = 2)
    Subnet[TtF4d2QWbk5vzQGTEPrN48x6vwgAoAmKQ9cbp79inpQmcRKES]: Validator Set: (Size = 2, Weight = 9223372036854775807)
    Validator[0]: NodeID-111111111111111111116DBWJs, 1\n
    Validator[1]: NodeID-QLbz7JHiBTspS962RLKV8GndWFwdYhk6V, 9223372036854775806
    Subnet[2mcwQKiD8VEspmMJpL1dc7okQQ5dDVAWeCBZ7FWBFAbxpv3t7w]: Validator Set: (Size = 1, Weight = 1)
    Validator[0]: NodeID-QLbz7JHiBTspS962RLKV8GndWFwdYhk6V, 1
```
Seems much less clean than:
```
Validator Manager: (Size = 2)
    Subnet[TtF4d2QWbk5vzQGTEPrN48x6vwgAoAmKQ9cbp79inpQmcRKES]: Validator Set: (Size = 2, Weight = 9223372036854775807)
        Validator[0]: NodeID-111111111111111111116DBWJs, 1\n
        Validator[1]: NodeID-QLbz7JHiBTspS962RLKV8GndWFwdYhk6V, 9223372036854775806
    Subnet[2mcwQKiD8VEspmMJpL1dc7okQQ5dDVAWeCBZ7FWBFAbxpv3t7w]: Validator Set: (Size = 1, Weight = 1)
        Validator[0]: NodeID-QLbz7JHiBTspS962RLKV8GndWFwdYhk6V, 1
```
Because the `Validator` index is part of the `Subnet` group